resources/config/varnish/fos_purge.vcl

```diff
@@ -12,6 +12,17 @@ sub fos_purge_recv {
         if (!client.ip ~ invalidators) {
             return (synth(405, "Not allowed"));
         }
+
+        # If http header "xkey" is set, we assume purge is on key and you have Varnish xkey installed
+        if (req.http.xkey) {
```

Comment:
note: I'm using `xkey` here, however xkey doc suggests using `key`, or in older doc `xkey-purge`. 

As it only supports 1 key here, it can make sense to use a different header, however then we need a separate config option in `src/ProxyClient/Varnish.php`.